webapp/src/index.js

```diff
@@ -24,10 +24,13 @@ class PluginClass {
     async initialize(registry, store) {
         registry.registerReducer(Reducer);
 
+        const {data: settings} = await getSettings(store.getState);
         await getConnected(true)(store.dispatch, store.getState);
 
-        registry.registerLeftSidebarHeaderComponent(SidebarHeader);
-        registry.registerBottomTeamSidebarComponent(TeamSidebar);
+        if (settings && settings.left_sidebar_enabled) {
```

Comment:
If I am not mistaken, this will not change whether it shows or not on runtime. A user would have to refresh the client to see the changes. Am I right?
@aaronrothschild Is this the behaviour we want?

Reply:
Yes,user has to refresh to see the changes.

Reply:
Ideally, the user would click on it and the settings would take place without a refresh on the client side.  Is this possible?  

In our desktop client, most users would not be familiar with how to refresh the page.

Reply:
@aaronrothschild Since this setting can only be changed by the system admin, the way I see this setting is more of a one-time setting, and it won't change regularly.

Reply:
If we still want the changes to reflect immediately , I can think regarding that.

Reply:
@jatinjtg sorry for the delayed response on this.  I thought I had posted a comment earlier.  

If this is just for admin when setting this - that should be ok.